charms/istio-pilot/README.md

```diff
@@ -27,6 +27,28 @@ juju relate istio-pilot:certificates <TLS certificates providers>:certificates
 
 > Please refer to the official documentation for more details about the [TLS ingress gateway for a single host](https://istio.io/latest/docs/tasks/traffic-management/ingress/secure-ingress/#configure-a-tls-ingress-gateway-for-a-single-host).
 
+## Enable the Istio CNI plugin
```

Comment:
tiny nit: can we add a link either in the title or in the first few sentences that links to istio's CNI docs explaining this feature?  I think someone who doesn't know this feature will want that extra context

Reply:
Updated